schoolopy/authentication.py

```diff
@@ -56,7 +56,9 @@ def _request_header(self):
             'Content-Type': 'application/json'
         }
 
-    def request_authorization(self):
+    def request_authorization(self, callback_url=None):
+        if callback_url == None:
+            callback_url = self.DOMAIN_ROOT
```

Comment:
Why not just include this in the function param? `...(self, callback_url=DOMAIN_ROOT):`

Reply:
Update: forgot that's dynamically set, this looks good